ledger_device_sdk/src/ecc.rs

```diff
@@ -403,6 +403,19 @@ pub fn bip32_derive(curve: CurvesId, path: &[u32], key: &mut [u8]) -> Result<(),
     Ok(())
 }
 
+/// Wrapper for 'os_perso_derive_eip2333'
+pub fn eip2333_derive(path: &[u32], key: &mut [u8]) -> Result<(), CxError> {
+    unsafe {
+        os_perso_derive_eip2333(
```

Comment:
I know it's what we have done already, but shouldn't we change it to use the no_throw version of the syscalls / cx_calls?

Reply:
I'm not sure, they are static inline function defines in os_perso.h @yhql  maybe it can't be used to generate bindings?